### PR TITLE
Ignore missing kinds when uninstalling

### PR DESCRIPTION
**Pull request by stefanprodan**
Ignore missing kinds when uninstalling, do not assume `bootstrap` was used.

Fix: #359

### CODE REVIEW

File: cmd/gotk/uninstall.go [hunk 1]
```diff
@@ -83,37 +83,55 @@ func uninstallCmdRun(cmd *cobra.Command, args []string) error {
...
+	err = kubeClient.Get(ctx, releaseName, &release)
+	if err == nil {
+		release.Spec.Suspend = true
+		if err := kubeClient.Update(ctx, &release); err != nil {
+			return fmt.Errorf("unable to suspend release '%s': %w", releaseName.String(), err)
+		}
+	}
+	if err == nil || apierrors.IsNotFound(err) {
+		namespacedKinds = append(namespacedKinds, helmv2.HelmReleaseKind)
+	}
```
Comment (hiddeco) [inline cmd/gotk/uninstall.go:128]: Given we do not bootstrap using a `HelmRelease` ourselves, this name assumption is likely wrong and the code thus obsolete?

Comment (stefanprodan) [reply cmd/gotk/uninstall.go:128]: The code tests if the HelmRelease Kind exists, should I remove the suspend update?

Comment (hiddeco) [reply cmd/gotk/uninstall.go:128]: Yes, I think that would be less confusing (at least to me).

**Review by hiddeco — APPROVED** [review]
Much easier to follow :100:

**Pull request merged**